.clj-kondo/hooks/clojure/test.clj

```diff
@@ -0,0 +1,46 @@
+(ns hooks.clojure.test
+  (:require [clj-kondo.hooks-api :as hooks]))
+
+(def disallowed-parallel-forms
+  '#{with-redefs
+     clojure.core/with-redefs
+     metabase.test/with-temporary-setting-values
+     mt/with-temporary-setting-values})
```

Comment:
What about `alter-var-root` and functions using that like `driver/register!`?

Reply:
Yeah, this should have wayyy more things in it than just these. But I ran into https://github.com/clj-kondo/clj-kondo/issues/1996 while working on this so I thought I'd just include what I had and pick this back up once that issue was fixed upstream